bin/lint_slugs.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+function check_slugs() {
+    duplicate_slugs=$(jq -r ".[\"$1\"][].slug" config.json | sort | uniq -d)
```

Comment:
One teeny jq suggestion:

```suggestion
    duplicate_slugs=$(jq -r --arg key $1 ".[$key][].slug" config.json | sort | uniq -d)
```